src/Workleap.Extensions.Mongo.Tests/MongoIndexerTests.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Workleap.Extensions.Mongo.Tests;
 
-public sealed class MongoIndexerTests : BaseIntegrationTest<MongoFixture>
+public partial class MongoIndexerTests : BaseIntegrationTest<MongoFixture>
```

Comment:
I believe it increased readability to split this class into partial classes. The whole class is about indexing.